arch/src/x86_64/interrupts.rs

```diff
@@ -13,13 +13,15 @@ use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
 use kvm_bindings::kvm_lapic_state;
 use kvm_ioctls::VcpuFd;
 
+/// Errors thrown while tackling the LAPIC.
```

Comment:
Generally tackling is used against something problematic. IMO it would be more appropriate to change this line to something like this:
```suggestion
/// Errors thrown while configuring the LAPIC.
```

Reply:
Done.